docs/examples/qr-camera/index.html

```diff
@@ -63,10 +71,56 @@ <h1 class="title">Scan QR Code from Video Camera</h1>
 
   <script type="text/javascript" src="https://unpkg.com/@zxing/library@latest"></script>
   <script type="text/javascript">
+    function decode_once(codeReader, selectedDeviceId) {
+      codeReader.decodeFromInputVideoDevice(selectedDeviceId, 'video').then((result) => {
+        console.log(result)
+        document.getElementById('result').textContent = result.text
+      }).catch((err) => {
+        console.error(err)
+        document.getElementById('result').textContent = err
+      })
+    }
+
+    function decode_continuously(codeReader, selectedDeviceId) {
+      codeReader.decodeFromInputVideoDeviceContinuously(selectedDeviceId, 'video', (result, err) => {
+        if (result) {
+          // properly decoded qr code
+          console.log(result)
+          document.getElementById('result').textContent += 'Found QR Code: ' + result.text + '\n\n'
+        }
+
+        if (err) {
+          // As long as this error belongs into one of the following categories
+          // the code reader is going to continue as excepted. Any other error
+          // will stop the decoding loop.
+          //
+          // Excepted Exceptions:
+          //
+          //  - NotFoundException
+          //  - ChecksumException
+          //  - FormatException
+
+          // Normally you should try to log or handle the error in here but in
+          // the current state of `decodeFromInputVideoDeviceContinuously` it's
+          // not worth tryin from pure JS.
+          //
+          // Main Reasons are:
+          //  1) There is no timeout after a decoding attemp failed so this path
+          //     of the callback will be called quite often. This might change in
+          //     the near future if zxing-js/library#187 will be merged.
+          //  2) Most of the errors don't contain a useful error message as they
+          //     are just handled by type inside typescript. This results in
+          //     a situation where you will only receive almost useless empty
```

Comment:
You will receive object instances typed as the error instances they are, JS does have a typing system, but it's dynamic, while TS introduces static typing syntax. So if you try `err instanceof NotFoundException` it will give you a meaningful result.

/cc @0ndorio